nifi-nar-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-restapi-processors/src/main/java/org/apache/nifi/processors/elasticsearch/AbstractJsonQueryElasticsearch.java

```diff
@@ -281,8 +342,25 @@ private void handleAggregations(final Map<String, Object> aggregations, final Pr
         }
     }
 
+    @SuppressWarnings("unchecked")
+    private Map<String, Object> formatAggregations(final Map<String, Object> aggregations) {
+        final Map<String, Object> formattedAggregations;
+
+        if (AGGS_METADATA_ONLY.getValue().equals(aggregationFormat)) {
+            formattedAggregations = new HashMap<>(aggregations);
```

Comment:
Recommend using `LinkedHashMap` for more deterministic results.
```suggestion
            formattedAggregations = new LinkedHashMap<>(aggregations);
```